woocommerce-services.php

```diff
@@ -897,6 +899,14 @@ public function woocommerce_payment_gateways( $payment_gateways ) {
 			return $payment_gateways;
 		}
 
+		public function woocommerce_stripe_request_headers( $headers ) {
+			$stripe_settings = get_option( 'woocommerce_stripe_settings', array() );
+			if ( isset( $stripe_settings['connect'] ) && 'yes' === $stripe_settings['connect'] ) {
+				unset( $headers['User-Agent'] );
+			}
+			return $headers;
+		}
+
```

Comment:
Sorry for coming late with this opinion, but let's keep the Stripe functionality in the `WC_Connect_Stripe` class.